cluster/gce/gci/configure-helper.sh

```diff
@@ -485,13 +485,16 @@ function create-master-audit-policy {
   local -r known_apis='
       - group: "" # core
       - group: "admissionregistration.k8s.io"
+      - group: "apiextensions.k8s.io"
+      - group: "apiregistration.k8s.io"
       - group: "apps"
       - group: "authentication.k8s.io"
       - group: "authorization.k8s.io"
       - group: "autoscaling"
       - group: "batch"
       - group: "certificates.k8s.io"
       - group: "extensions"
+      - group: "metrics"
```

Comment:
should be `metrics.k8s.io`

Reply:
As discussed offline, I'll leave it as it is in this PR and @piosz will fix it together with changing the names in the rest of the repo